website/docs/r/launch_configuration.html.markdown

```diff
@@ -1,4 +1,5 @@
 ---
+subcategory: "EC2"
```

Comment:
Not for this pull request, but this is confusingly in Autoscaling for the data source and EC2 for the resource. Technically, I think it should be in Autoscaling since that's the service API.